props/saber_BC_buttons.h

```diff
@@ -463,7 +463,23 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
       } else {
         thrust_begin_millis_ = millis();
       }
-  }
+
+    // Scroll Presets timer check - avoid beep/wav overlap
+    if (scroll_presets_timer_trigger_ && DelayTimer(scroll_presets_timer_trigger_, last_timer_time_, beep_seq_duration_ * 1000)) {
```

Comment:
What's the point of using a float for beep_seq_duration_ if you're just going to use it as millis anyways?